package/tests/test_server.py

```diff
@@ -46,117 +47,20 @@
 from kedro_viz.server import _allocate_port, _sort_layers, format_pipelines_data
 from kedro_viz.utils import WaitForException
 
-EXPECTED_PIPELINE_DATA = {
```

Comment:
I had to move this somewhere because the linter was complaining that `test_server.py` exceeded 1000 lines. Nothing in `EXPECTED_PIPELINE_DATA` was changed, it's just copy/paste.